en/components/combo_cascading.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Cascading Scenario
+_description: The igx-combo and igx-drop-down exposes events, properties and methods that allows to easily construct cascading scenario using one of the controls or both of them.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Combo components, Angular Combo controls, Angular Cascading Combo, Angular Drop Down, Angular Cascading Scenario
+---
+
+## Cascading Scenario
+<p class="highlight">
+The igx-combo and igx-drop-down exposes events, properties and methods that allows to easily construct cascading scenario, either using one of the controls or both of them.
+</p>
+<div class="divider"></div>
+
+### Demo
+
+The following sample demonstrates scenario, using two igx-drop-down and igx-combo components:
+
+<div class="sample-container loading" style="height: 540px;">
+    <iframe id="cascading-combos-sample" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/cascading-combos" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="cascading-combos-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+### Usage
+To get started with the Ignite UI for Angular Combo import the **IgxComboModule** in the **app.module.ts** file:
```

Comment:
In the context of this example, we should let the user he needs to also include IgxDropDownModule, IgxIconModule and IgxInputGroupModule in the imports.